projects/mercury/src/file/FileList.js

```diff
@@ -49,6 +50,7 @@ export const FileList = ({
 
     const [filterValue, setFilterValue] = useState("");
     const [filteredFiles, setFilteredFiles] = useState(files);
+    const {vocabulary} = useContext(VocabularyContext);
```

Comment:
better to fetch elements from the context at the wrapping component level (ContextualFileList), same as done for hierarchy (line 242), otherwise the tests are failing